lib/terminal-table/table.rb

```diff
@@ -1,4 +1,4 @@
-require 'unicode/display_width/no_string_ext'
+require 'unicode/display_width'
```

Comment:
This file does not exist in v1.1.1.

Reply:
I'm not sure I follow, could you elaborate? I see https://github.com/janlelis/unicode-display_width/blob/v1.1.1/lib/unicode/display_width.rb

Also, if i pin the dependency to v1.1.1, then `rake spec` runs without issue.

Reply:
LOL, ignore me, no idea what I was looking at